src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerCreateForm.tsx

```diff
@@ -46,9 +47,10 @@ type TemplatesCatalogDrawerCreateFormProps = {
 export const TemplatesCatalogDrawerCreateForm: FC<TemplatesCatalogDrawerCreateFormProps> = memo(
   ({ authorizedSSHKey, namespace, onCancel, subscriptionData }) => {
     const { t } = useKubevirtTranslation();
+    const cluster = useClusterParam();
 
     const { isBootSourceAvailable, templateLoadingError } = useDrawerContext();
-    const [__, vmsNotSupported] = useNamespaceUDN(namespace);
+    const [__, vmsNotSupported] = useNamespaceUDN(namespace, cluster);
```

Comment:
Why are we passing the cluster into hooks instead of calling `useClusterParam` inside the hooks directly? We should be minimizing instances of passing the cluster around as much as possible, IMO.